src/test/java/uk/gov/justice/digital/delius/service/ConvictionServiceTest.java

```diff
@@ -29,7 +30,7 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-@RunWith(SpringRunner.class)
+@ExtendWith(SpringExtension.class)
```

Comment:
Refactored to JUnit 5 and introduced nested hence the huge change - really just moved tests into appropriate nested tests